src/app/services/table.service.ts

```diff
@@ -0,0 +1,53 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+
+export class TableService {
+
+  constructor() { }
+
+  users = [
```

Comment:
ta część do poprawienia jeszcze, trzeba dodać obsługę z rxjs w tym przypadku funkcji `of` :)